src/python/common/tool_arguments.py

```diff
@@ -0,0 +1,72 @@
+from dataclasses import dataclass
+from enum import Enum, unique
+from typing import List, Union
+
+from src.python.review.inspectors.inspector_type import InspectorType
+
+
+@unique
+class VerbosityLevel(Enum):
+    """
+    Same meaning as the logging level. Should be used in command-line args.
+    """
+    DEBUG = '3'
+    INFO = '2'
+    ERROR = '1'
+    DISABLE = '0'
+
+    @classmethod
+    def values(cls) -> List[str]:
+        return [member.value for _, member in VerbosityLevel.__members__.items()]
+
+
+@dataclass(frozen=True)
+class ArgumentsCharacteristics:
+    short_name: Union[str, None]
+    long_name: str
+    description: str
+
+
+inspectors = [inspector.lower() for inspector in InspectorType.available_values()]
+example = f'-d {inspectors[0].lower()},{inspectors[1].lower()}'
+
+
+@unique
+class RunToolArguments(Enum):
+    VERBOSITY = ArgumentsCharacteristics('-v',
+                                         '--verbosity',
+                                         'Choose logging level: '
+                                         f'{VerbosityLevel.ERROR.value} - ERROR; '
+                                         f'{VerbosityLevel.INFO.value} - INFO; '
+                                         f'{VerbosityLevel.DEBUG.value} - DEBUG; '
+                                         f'{VerbosityLevel.DISABLE.value} - disable logging; '
+                                         'default is 0')
+
+    DISABLE = ArgumentsCharacteristics('-d', '--disable',
+                                       'Disable inspectors. '
+                                       f'Allowed values: {", ".join(inspectors)}. '
+                                       f'Example: {example}')
+
+    DUPLICATES = ArgumentsCharacteristics(None, '--allow-duplicates',
+                                          'Allow duplicate issues found by different linters. '
+                                          'By default, duplicates are skipped.')
+
+    LANG_VERSION = ArgumentsCharacteristics('--language_version', '--language_version',
+                                            'Specify the language version for JAVA inspectors.')
+
+    CPU = ArgumentsCharacteristics('--n_cpu', '--n-cpu',
```

Comment:
I think it should just be replaced by this: `CPU = ArgumentsCharacteristics(None, '--n-cpu',`.  And then in `run_tool`, when adding this argument, add `--n_cpu` next to it.